nextflow.config

```diff
@@ -18,6 +18,7 @@ params {
     igenomes_ignore            = false
     local_genomes              = null
     save_reference             = false
+    mt_contig_name             = 'MT'
```

Comment:
It's been a bit back and forth but I think the default should be running with grch38 and then the default name should be `chrM`

Reply:
Yeah, sure. While we are not running grch38 ourselves, I imagine by the time we deploy this in production we will be there (hopefully) 🤞🏻 😄